### DISCUSSION TITLE
Why can't I be verified for GitHub campus?

### DISCUSSION THREAD

**Discussion by Chirigiu** [community/community#52333] [GitHub Education]
On the email I get "You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.". And this is the picture that I sent, I hided my cnp and phone number here, but I sent them to the application with them showing:
Moderator Edit: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

My name is Chirigiu Stefan-Alexandru as it appears in the photo that I sent, that is the exact same name on my profile and I gave a short bio about my school and the place where I live too, so why does my appeal gets automatically rejected if the data on the photo and on my profile matches?

**Comment by samus-aran** [comment]
Hi @Chirigiu 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. Try to know skew your image when you upload it or taking the photo.

Let us know how you get on 👍
